README.md

```diff
@@ -34,4 +34,17 @@ client.waitForInitialization();
 final boolean bResult = client.boolVariation("bool-flag", null, false);
 
 assert bResult == true
-```
\ No newline at end of file
+```
+* flag: demo-flag
+
+# Flags
+
+Application Options:
+-t, --timeout=     Request timeout
+-s, --status-code= returns HTTP status code
+-m, --message=     Message to display in response
+-e, --sse=         SSE off sequence, -e=10 -e=30 -e=60 means it will go off in 10s, 30s and 60s
+-o, --operation=   operation
```

Comment:
I think it would be worthwhile explaining what the operation's are and where you can find a list of valid ones.